phpunit.xml

```diff
@@ -1,14 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
-
-<phpunit bootstrap="vendor/autoload.php" colors="true">
-    <testsuites>
-        <testsuite name="All tests">
-            <directory>tests</directory>
-        </testsuite>
-    </testsuites>
-    <filter>
-        <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">./src</directory>
-        </whitelist>
-    </filter>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="vendor/autoload.php" colors="true" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
```

Comment:
Why does this need to be replaced? Does previous configuration have anything wrong?

Reply:
No different in run test results.

I added `xmlns:xsi` and `xsi:noNamespaceSchemaLocation` when version up phpunit too.
Official phpunit.xml wrote these property. that cound be make more strict.

```
https://github.com/sebastianbergmann/phpunit/blob/master/phpunit.xml
```